azerothcore/azerothcore-wotlk
# Issue 1238

##### CHANGES PROPOSED:

-  Rewrote the logic to retrieve data from the DB


###### ISSUES ADDRESSED:


Closes #1238 


##### TESTS PERFORMED:

The build is without errors, tested in game after create a new account at lvl 1. Tested on windows 10



##### HOW TO TEST THE CHANGES:

- Close all tutorial panels, logout/exit game (the new row is in db) and no panels show on next login
- In settings panel disabled the "Show tutorial" and no panels show on next login
- When the tutorial is no longer shown, in settings panel reset the tutorial and the row in db is updated, panel are shown istant and persit if no closed


##### KNOWN ISSUES AND TODO LIST:


##### Target branch(es):

Master
